app/package.json

```diff
@@ -38,7 +38,8 @@
     "redux-persist": "6.0.0",
     "redux-saga": "1.1.3",
     "redux-sagas-injector": "1.1.1",
-    "selectorator": "4.0.3"
+    "selectorator": "4.0.3",
+    "todoist-rest-api": "^1.3.4"
```

Comment:
A second, more generic comment: I've started the project [without any server at all](https://dashboard.darekkay.com/docs/blog/2020-04-11.html#server-module), thinking I could go as far as possible with that approach. However, I've now implemented the first 3rd-party-data-driven widgets (Cryptocurrencies, GitHub Stats). Now I think we should handle all 3rd-party data via the server module for the following reasons:

1. [Request caching](https://dashboard.darekkay.com/docs/blog/2020-04-22.html#request-caching). If the 3rd-party request is implemented on the client, the Todoist API will be hit directly every time the user refreshes the page (page refresh overrides the widget update cycle).

2. When we switch to OAuth later, a developer key will be required. If the request doesn't go through the server module, the developer key will be exposed to the users, which it shouldn't.

In other words, I would move this module from `app/package.json` to `server/package.json` and create internal `/todoist` route(s) as a proxy/facade.

The server module is only a few weeks old, but there are already 3 different routes implemented (2 of them actually used) that you can check as an example.